pages/index.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: home
-tagline: Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the federal government
+tagline: Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the Federal government
```

Comment:
Federal Enterprise Data Resources are tools, best practices, and schema standards to facilitate the adoption of open data practices across the federal government.